doc/source/missing_data.rst

```diff
@@ -330,6 +330,10 @@ Interpolation
 
   The ``limit_direction`` keyword argument was added.
 
+.. versionadded:: 0.21.0
+
```

Comment:
A link to below sounds good. You can make a new one specifically for `_missing_data.interp_limit_area`